lighthouse-core/audits/seo/tap-targets.js

```diff
@@ -18,6 +18,29 @@ const {
   getLargestRect,
 } = require('../../lib/rect-helpers');
 const {getTappableRectsFromClientRects} = require('../../lib/tappable-rects');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on whether tap targets (like buttons and links) on a page are big enough so they can easily be tapped on a mobile device. This descriptive title is shown when tap targets are easy to tap on. */
+  title: 'Tap targets are sized appropriately',
+  /** Title of a Lighthouse audit that provides detail on whether tap targets (like buttons and links) on a page are big enough so they can easily be tapped on a mobile device. This descriptive title is shown when tap targets are not easy to tap on. */
+  failureTitle: 'Tap targets are not sized appropriately',
+  /** Description of a Lighthouse audit that tells the user why buttons and links need to be big enough and what 'big enough' means. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'Interactive elements like buttons and links should be large enough (48x48px), and have enough space around them, to be easy enough to tap without overlapping onto other elements. [Learn more](https://developers.google.com/web/fundamentals/accessibility/accessible-styles#multi-device_responsive_design).',
+  /** Label of a table column that identifies tap targets (like buttons and links) that have failed the audit and aren't easy to tap on. */
+  tapTargetHeader: 'Tap Target',
+  /** Label of a table column that specifies the size of tap targets like buttons and links. */
+  sizeHeader: 'Size',
+  /** Label of a table column that identifies a tap target (like a link or button) that overlaps with another tap target. */
+  overlappingTargetHeader: 'Overlapping Target',
+  /** Explanatory message stating that there was a failure in an audit caused by the viewport meta tag not being optimized for mobile screens, which caused tap targets like buttons and links to be too small to tap on. */
+  // eslint-disable-next-line
+  explanationViewportMetaNotOptimized: 'Tap targets are too small because there\'s no viewport meta tag optimized for mobile screens',
+  /** Explanatory message stating that a certain percentage of the tap targets (like buttons and links) on the page are of an appropriately large size. */
+  displayValue: '{decimalProportion, number, percent} appropriately sized tap targets',
```

Comment:
`percent` is a default iirc, it just prunes decimals which is why we needed `extendedPercent`.  As long as we are okay with no decimals then we should be good with percent here.  Might need to add this to the LR tc bundler though, since I don't think we've ever used just `percent` so far.

Reply:
> Might need to add this to the LR tc bundler though, since I don't think we've ever used just percent so far.

What does this mean, and is it something I have to do?

Reply:
Anything left for me to do here? Or is this PR good to go?

Reply:
AFAICT, we never actually flex this in the tests, can we add an assertion for it looking like we expect? then I'm good :)

off to @exterkamp on the "LR tc" bit since I'm not 100% where that change would be made either